.gitignore

```diff
@@ -1,10 +1,13 @@
-work/
-.idea/
-target/
-test-output/
-.idea/
-.classpath
-.settings/
-.project
-**/*~
-*.iml
+work/
+.idea/
+target/
+test-output/
+.classpath
+.settings/
+.project
+**/*~
+*.iml
+node/
+node_modules/
+src/main/resources/org/biouno/unochoice/stapler/unochoice/*.js
+src/main/resources/org/biouno/unochoice/stapler/unochoice/*.js.map
```

Comment:
I think that means webpack is writing the ES5 generated files into the Java source directory. Isn't it possible to write it directly to the `target` dir, and avoid having to ignore build-generated files in the sources dirs?

Reply:
This is done now. And it works as expected.